client/apps/admin-portal/src/app/routing.module.ts

```diff
@@ -4,12 +4,16 @@ import { SermonsComponent } from '@sb/sermons';
 import { ManageComponent } from './manage/manage.component';
 
 const routes: Routes = [
+  {
```

Comment:
The `ManageComponent` does not need two routes.

Routes should be as follow:
- `''`: SermonsComponent
- `'manage'`: ManageComponent